scripts/prepare-licenses.sh

```diff
@@ -27,8 +27,6 @@ licenseRegex="\(LICEN\(S\|C\)E\|COPYING\|README\|NOTICE\)"
 licenses="${tmp}"/LICENSES.txt
 
 # Ensure the vendor directory only includes the project's build dependencies.
-go mod tidy -compat=1.17
-go mod vendor
```

Comment:
historically, the vendor was required here to make sure the licenses were in the vendor dir to aggregate together. However, now that the vendor dir is committed in the repo, we do need to make sure we do a go mod vendor before aggregating licenses and before checking for no diffs. So if we remove it here, it needs a top level make target to be a dependency.

Reply:
It is run as part of the `tidy` target

Reply:
Ahh. I missed that... I don't know if that makes sense to me. I would expect a different `make vendor` to do that and then use tidy and vendor targets as needed...

But if you want one to do both, maybe name it `make deps`?